src/modals.js

```diff
@@ -175,14 +163,6 @@ function NewProjectModalCancelClick() {
             // If there is no project, go to home page.
             window.location.href = 'index.html' + window.getAllURLParameters();
         }
-        // Reload the the editor canvas from the active copy of the
```

Comment:
Note to self. This code is now delegating the handling of project state off to code in the editor.js file. There, the onDocumentLoad event handler is checking to see if the projectData object has a project. If it does not, as in the case of a page transition, it will retrieve the project from the browser local store. This whole operation happens in milliseconds, so it appears to be seamless from the user perspective.